pages/index.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+	<div>
+		<a-row type="flex" justify="center">
+			<a-col :span="12">Wallet Connet</a-col>
+		</a-row>
+		<a-row type="flex" justify="center">
+			<a-col :span="12">
+				<a-dropdown>
+					<template #overlay>
+						<a-menu @click="handleMenuClick">
+							<a-menu-item key="1"> Algosigner </a-menu-item>
+							<a-menu-item key="2"> Wallet Connect </a-menu-item>
+							<a-menu-item key="3"> My Algo Wallet </a-menu-item>
+						</a-menu>
+					</template>
+					<a-button>
+						Select your wallet
+						<DownOutlined />
+					</a-button>
+				</a-dropdown>
```

Comment:
please move the wallet select / connect to a separate component (eg `/components/wallet.vue`)

Reply:
Done